src/encoded/search.py

```diff
@@ -352,13 +352,13 @@ def get_all_subsequent_results(initial_search_result, search, extra_requests_nee
             yield hit
 
 def execute_search_for_all_results(search):
-    size_increment = 100 # Decrease this to like 5 or 10 to test.
+    size_increment = 100  # Decrease this to like 5 or 10 to test.
```

Comment:
I know you didn't create this variable, but while we're in here: I have a mild preference for the variable name `chunk_size` for this. You're not incrementing something, you're just dividing it into chunks.